NYSERDA

hawbecker opened the pull request:
Adding required files for NYSERDA documentation.

hawbecker committed:
Modified for ERA5

hawbecker committed:
Adding new files for FINO

hawbecker committed:
Merge branch 'master' of https://github.com/phawbeck/WRF-setups

hawbecker committed:
Merge branch 'master' of https://github.com/a2e-mmc/WRF-setups

hawbecker committed:
Adding files for NYSERDA case study documentation

hawbecker committed:
Replacing username with $USER

hawbecker committed:
Reverting back to original

hawbecker committed:
Updating README to README.md

hawbecker committed:
Making cleaner for MD

lassman135 reviewed (commented):
I'm not sure I understand...did you comment out runniing 'link_grib' for ERA5? Is this because the ERA5 is processed through a different mechanism than Ungrib? I remember having a conversation about this, just wanted to double-check before I approve.

hawbecker committed:
Modifying workflow and namelist vars

The CPM utility has been updated so the changes are reflecting
the new namelist variables. Also, the workflow for creating the
tslist file required wrfinput_d05 which is not created until
after real.exe, so, I changed it to use a met_em file and moved
this until after the files are linked. This should make things
more streamlined.

hawbecker committed:
Adding modified namelists

Changes were made in MMC-WRF for the CPM and depth-dependent
roughness. The namelist changes reflect these changes. Also, a
line for testing purposes was removed from the setup script.
